src/utils/userAgent.js

```diff
@@ -1,9 +1,10 @@
 /* @flow strict-local */
 import { NativeModules } from 'react-native';
 import DeviceInfo from 'react-native-device-info';
+import { nativeApplicationVersion } from 'expo-application';
 
-const { getVersion, getSystemName, getSystemVersion } = DeviceInfo;
+const { getSystemName, getSystemVersion } = DeviceInfo;
 
 export default !NativeModules.RNDeviceInfo
```

Comment:
Huh, this conditional is odd. Dates back to 9d9720c74ccf0aebb35a51e018ced676dbc7970a.

Reply:
Thanks for the diagnosis and cleanup!